metals/src/main/scala/scala/meta/internal/metals/BuildTargets.scala

```diff
@@ -52,8 +53,6 @@ final class BuildTargets(
     TrieMap.empty[AbsolutePath, BuildTargetIdentifier]
   private val isSourceRoot =
     ConcurrentHashSet.empty[AbsolutePath]
-  private val buildTargetInference =
```

Comment:
This was not actually used anywhere. `sourceItemsToBuildTarget` is used for the same exact purpose.